projects/admin/src/app/service/record-routing.service.ts

```diff
@@ -279,6 +280,17 @@ export class RecordRoutingService {
     });
   }
 
+  /**
+   * Parse url with pid from $ref
+   * @param record - object, record to be saved
+   */
+  private getUrl(record: any) {
+    const pidRegExp = new RegExp('.*/(.*)$');
```

Comment:
Use `extractIdOnRef` to extract the pid

Reply:
`extractIdOnRef ` doesn't use the same regex. I don't think it does the right job.

Reply:
I use this code on another redirect route:
```
const budgetPid = extractIdOnRef(record.budget.$ref);
return of(`records/budgets/detail/` + budgetPid);
```

Reply:
From $ref = ils.rero.ch/api/documents/286, it gives pid = 6 (tested with https://rubular.com/)